### ISSUE TITLE
node-keyboard doesn't work on Mac OS 10.13.3

### ISSUE THREAD

**Issue by zhaoyi0113** [justinjmoses/node-keyboard#8]
I installed node-keyboard on Mac OS `10.13.3` but get below errors when typing anything on the terminal. I am using node v8.9.1, npm 5.5.1

```
node-keyboard>MIDI not detected. Disabling support.
fs.js:75
    throw new TypeError('"options" must be a string or an object, got ' +
    ^

TypeError: "options" must be a string or an object, got number instead.
    at getOptions (fs.js:75:11)
    at Object.fs.writeFile (fs.js:1261:13)
    at ReadFileContext.fs.readFile [as callback] (/Users/joey/.nvm/versions/node/v8.9.1/lib/node_modules/node-keyboard/lib/index.js:42:23)
    at FSReqWrap.readFileAfterOpen [as oncomplete] (fs.js:420:13)
```

**Comment by jjgonecrypto** [comment]
Hi there. I just tried reinstalling on `node 8.93.` and `npm 5.8.0` on Mac OS `10.13.4` with no issues (I had to remove all `node_modules`  and `npm install` again though as the node-gyp modules had to be rebuilt. Can you try removing `node_modules` and trying again? If that fails, can you try with the latest node and npm (I tried with those also and no issues).

**Comment by zhaoyi0113** [comment]
Hi, thanks for your reply. I didn't install from source. I run `npm install -g node-keyboard` command to install `node-keyboard` globally. So there is no need to remove the node_modules right?

**Comment by jjgonecrypto** [comment]
That's correct. Let me try this later today and get back to you

**Comment by jjgonecrypto** [comment]
Fixed in 3.9.3 and published. Thanks for reporting @zhaoyi0113!
